anaconda.py

```diff
@@ -46,10 +46,10 @@ def exitHandler(rebootData):
 
     # pylint: disable=possibly-used-before-assignment
     # pylint: disable=used-before-assignment
-    if "nokill" in kernel_arguments:
+    if "inst.nokill" in kernel_arguments:
         util.vtActivate(1)
-        print("anaconda halting due to nokill flag.")
-        print("The system will be rebooted when you press Ctrl-Alt-Delete.")
+        log.info("Anaconda halting due to inst.nokill flag.")
```

Comment:
Actually the print to log.info changes are unrelated. Please do seperate commit if needed.